cmd/admin-subnet-health.go

```diff
@@ -178,37 +170,40 @@ func mainAdminHealth(ctx *cli.Context) error {
 	// Get the alias parameter from cli
 	aliasedURL := ctx.Args().Get(0)
 
-	license, schedule, dev, name := fetchSubnetUploadFlags(ctx)
+	license, schedule, name, offline := fetchSubnetUploadFlags(ctx)
+
+	uploadToSubnet := !offline
+	if uploadToSubnet && !subnetReachable() {
+		console.Fatalln("https://subnet.min.io is not reachable.")
```

Comment:
Is it possible to include the proxy information if provided? .e.g "SUBNET is not reachable using proxy <ADDRESS>" ?